### ISSUE TITLE
Update Proctypes

### ISSUE THREAD

**Issue by rmendelovits** [digiaonline/react-foundation#35]
### What did you do?
npm install --save react-foundation
### What did you expect to happen?
program would run
### What actually happened?
console.warn node_modules/react/lib/lowPriorityWarning.js:40
    Warning: Accessing PropTypes via the main React package is deprecated, and will be removed in  React v16.0. Use the latest available v15.* prop-types package from npm instead. For info on usage, compatibility, migration and more, see https://fb.me/prop-types-docs
### What version of this module are you using?
"react-foundation": "^0.8.0",
Please include code that reproduces the issue. The best reproductions are self-contained scripts with minimal dependencies.

```
import { Badge, Colors } from 'react-foundation';
...
...
          <div className="badge-colors-example">
            <Badge color={Colors.SECONDARY}>2</Badge>
            <Badge color={Colors.SUCCESS}>3</Badge>
            <Badge color={Colors.ALERT}>A</Badge>
            <Badge color={Colors.WARNING}>B</Badge>
          </div>
```

**Comment by larpo** [comment]
This is duplicate of #31 and is fixed with #34

**Comment by rmendelovits** [comment]
thank you
